src/app/components/datetime-picker/datetime-picker.component.ts

```diff
@@ -16,7 +17,17 @@ import { getLangKeyString } from '../../modules/common-core/models/lang-key.enum
  * The date and time picker is displayed in a modal, and the selected date and time is returned to the parent component.
  */
 export class DatetimePickerComponent implements OnInit {
-  @Input() dateTime: string; // Supports Date.prototype.toISOString() format (YYYY-MM-DDTHH:mm:ss.sssZ)
+  // Supports Date.prototype.toISOString() format (YYYY-MM-DDTHH:mm:ss.sssZ)
+  // We need datePipe.transform to show the current date in the input node on web
+  _dateTime: string;
+  @Input() set dateTime(value: string) {
+    this._dateTime = this.datePipe.transform(value, 'yyyy-MM-ddTHH:mm');
+  }
+
+  get dateTime(): string {
+    return this._dateTime;
+  }
```

Comment:
Det er forvirrende at du bruker forskjellige properties for dato-verdien på web og app, særlig når navnet er nesten likt og ikke sier noe om hva som brukes til hva.
Kan du ikke ha en felles property og heller sjekke for platform inni setter eller getter som konverterer mellom web-format og app-format?